static/src/stylesheets/module/content/_article-immersive.scss

```diff
@@ -102,6 +102,7 @@
         .content__head--mobile {
             position: relative;
             height: 100vh;
+            height: -webkit-calc(100vh - 48px); // To compensate for mobile Safari viewport height issue.
```

Comment:
There's no guarantee that browsers will not implement/remove these declarations in the future, thus changing the result of this. But I don't have a better suggestion, so :+1: